build/gulp/js-bundles.js

```diff
@@ -56,8 +56,8 @@ gulp.task('js-bundles-prod', gulp.series('version-replace', function() {
 }));
 
 
-var createDebugBundlesStream = function(watch) {
-    var debugConfig, bundles;
+const createDebugBundlesStream = function(watch) {
+    let debugConfig; let bundles;
```

Comment:
It's better to declare variables from a new line.
Can we add a corresponding rule?

Reply:
I can add `one-var-declaration-per-line` rule. But I think it's better to do it in another PR.

Reply:
Just tried. The `one-var-declaration-per-line` is not what we need. The `newline-after-var` does not helped too. The `max-statements-per-line` is correct rule, but it could not be fixed automatically.

Reply:
Just replased it using the regular expression.
https://github.com/DevExpress/DevExtreme/pull/11318/commits/d1c5246eb96094c04b3a997ff7fdfe79d98d8b9b